services/alertmanager/alertmanager.go

```diff
@@ -43,20 +46,41 @@ const (
 
 	alertmanagerConfigPath     = "/etc/alertmanager.yml"
 	alertmanagerBaseConfigPath = "/srv/alertmanager/alertmanager.base.yml"
+
+	shippedRuleTemplatePath     = "/tmp/ia1/*.yml"
+	userDefinedRuleTemplatePath = "/tmp/ia2/*.yml"
+
+	envShippedRuleTemplatePath     = "PERCONA_TEST_SHIPPED_RULE_TEMPLATE_PATH"
+	envUserDefinedRuleTemplatePath = "PERCONA_TEST_USER_DEFINED_RULE_TEMPLATE_PATH"
```

Comment:
There is no need for them – QA will be able to put files into normal directories for users